Source/Mosa.Kernel.BareMetal/HardwareAbstractionLayer.cs

```diff
@@ -34,12 +36,7 @@ public override void Sleep(uint milliseconds)
 
 	public override void DebugWriteLine(string message) => Debug.WriteLine(message);
 
-	public override void Abort(string message)
```

Comment:
Let's keep as is - rather than revert to environment.

Reply:
Can't we at least disable all graphics devices when we abort?

Reply:
The abort (assuming debug or test is activated) will kill the VM.

Reply:
But usually there's a message attached to the abort, so it's be nice for the user to know about it.